packages/server/postgres/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM postgres:13.1
+ENV POSTGRES_PASSWORD password
```

Comment:
i'm always scared of bad defaults.
if we write `ENV POSTGRES_PASSWORD` it will let docker know we expect that var, but then we don't have a bad default hardcoded in there

Reply:
@mattkrick can you elaborate on what you mean by this comment?

Reply:
keep env vars in env file, not in code